lib/pharos/host/el7/scripts/upgrade-kubeadm.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+. /usr/local/share/pharos/util.sh
+
+set -ex
+
+if [ $(kubeadm version -o short) = "v${VERSION}" ]; then
+    exit
+fi
+
+versionlock="/etc/yum/pluginconf.d/versionlock.list"
+linefromfile "^0:kubeadm-" $versionlock
```

Comment:
Could these versionlock things be put into something generically re-usable? Now we have these lines repeating all over the scripts. Maybe some helper func like `yum_install <package> <version>` that handles all these things.

Reply:
Fixed.